.github/workflows/build.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       matrix:
         os: [ubuntu, macos]
         compiler: [g++, clang++]
-        defines: [standard, PUGIXML_WCHAR_MODE, PUGIXML_COMPACT, PUGIXML_NO_EXCEPTIONS]
+        defines: [standard, PUGIXML_WCHAR_MODE, PUGIXML_COMPACT, PUGIXML_NO_EXCEPTIONS, PUGIXML_STRING_VIEW]
```

Comment:
I am not familiar with appveyor -- please advise what changes you would recommend (if any) for that system.

Reply:
New define could be added to tests/autotest-appveyor.ps1 line 40. This by itself might not be enough due to C++17 restriction; Appveyor tests a variety of compiler versions most of which predate C++17 but that's fine as these would be tested via GitHub Actions.

Reply:
sounds good, I'll add a note about this in the description. Once the opt-in define is retired, appveyor builds will start exercising this automatically in the appropriate scenarios.